packages/core/chrome/core-chrome-browser-internal/src/ui/header/collapsible_nav.scss

```diff
@@ -24,3 +24,19 @@
     text-decoration: underline; /* 2 */
   }
 }
+
+/**
+ * Allows the CollapsibleNavGroups to be viewed on
+ * very small screen sizes and when the browser Zoom is high
+ */
+@media (max-height: 15em) {
+  .euiCollapsibleNav {
+    overflow-y: auto;
```

Comment:
Rather than applying this to **all** EuiCollapsibleNavs across Kibana, we should **only** be appplying this to this specific `.kbnCollapsibleNav` (applying the className if it's not already present).